packages/jest-editor-support/src/__tests__/parsers/parserTests.js

```diff
@@ -13,6 +13,9 @@ import type {ParserReturn} from '../../ScriptParser';
 const path = require('path');
 const fixtures = path.resolve(__dirname, '../fixtures');
 
+// silence the no-test error from Jest.
+test();
```

Comment:
this test should be ignored, so I'm not sure why you need to add this here. Can you remove it?